magento/magento2
# Skip is_scalar for null values when imploding multiline address

### Description (*)

A conditional is added to skip the is_scalar check for null values. Null values are treated as an empty string by implode() anyway. Allows a null value for the second street line in API requests.




### Fixed Issues (if relevant)

1. Fixes magento/magento2#35160

### Manual testing scenarios (*)

1. Create an empty guest cart with GraphQL.
![image](https://user-images.githubusercontent.com/39271781/174965107-4a1296c1-14cd-4192-abce-341d3aea5b1b.png)

2. Set the shipping address using the cart_id that was returned from the previous request (see [DevDocs](https://devdocs.magento.com/guides/v2.4/graphql/tutorials/checkout/checkout-shipping-address.html)). Use null as the second value in the street array.
![image](https://user-images.githubusercontent.com/39271781/174965377-2f828c60-77dd-4b1d-9c89-d876de49129a.png)


Before the change it will return `The address failed to save. Verify the address and try again.`, after the change it will work as usual.




### Contribution checklist (*)
